cmd/operator-sdk/generate/bundle/bundle_legacy.go

```diff
@@ -0,0 +1,131 @@
+// Copyright 2020 The Operator-SDK Authors
```

Comment:
Why do we need to create the legacy one? Is not it == the new one and just change the paths? 
Is not harder to have the same implementation duplicated instead of having the code centralized to keep it maintained?

Reply:
I'm of the opinion that some code duplication is OK, especially with complex command setup like this. It's easier to maintain, then remove when we remove current project layout support.

Reply:
I'm okay with duplication in this case as well. From my experience with trying to reuse the code for `generate csv` to be used by both the old and new CLI commands it made the underlying command a lot more convoluted.

@camilamacedo86 `operator-sdk run local` was a much simpler command where we just had to change 1 path.